extensions/big-data-cluster/src/bigDataCluster/dialog/hdfsDialogBase.ts

```diff
@@ -28,6 +28,12 @@ export interface HdfsDialogProperties {
 	password?: string;
 }
 
+export class HdfsDialogCancelledError extends Error {
+	constructor(message: string = 'Dialog cancelled') {
```

Comment:
Does this need localization?

Reply:
Depends. I generally go the route of "error messages shouldn't be exposed to the user and so don't need to be localized". But I also know we don't do that everywhere so if we think it's appropriate to default to localized errors I can switch it. I just don't want to make debugging a pain by having localized error messages in the console logs.